system-tests/_scripts/setup.sh

```diff
@@ -5,9 +5,9 @@ source .env/bin/activate
 
 # Install DC/OS CLI in the sandbox so we can setup and teardown our tests
 if [ `uname -s` == "Darwin" ]; then
-  curl https://downloads.dcos.io/binaries/cli/darwin/x86-64/dcos-1.11/dcos -o .env/bin/dcos
+  curl https://downloads.dcos.io/binaries/cli/darwin/x86-64/dcos-1.12/dcos -o .env/bin/dcos
```

Comment:
Shall we go with `https://downloads.dcos.io/binaries/cli/darwin/x86-64/dcos-latest/dcos` instead?
I wonder if we could extract this as ENV so that Jenkinsfile controls the version for CI and release scripts

Reply:
is `-latest` reliable? 🤔

Reply:
I'd assume if we test against `latest` cluster we should test along with `latest` CLI that way we could spot issues early enough